test/schemas/22-5490/schema.spec.js

```diff
@@ -14,7 +14,11 @@ const schemaDefaults = {
   }
 };
 
-let schemaTestHelper = new SchemaTestHelper(_.omit(schema, 'anyOf'), schemaDefaults);
+let liteSchema = _.cloneDeep(schema);
+delete liteSchema.anyOf;
```

Comment:
I only vaguely understand why we need to treat `anyOf` fields in our schemas differently. Can you add a comment here and below where you call `_.omit(schema, 'anyOf')` explaining the reason?